frontend/shared/components/src/organizations/components/SearchUitpasOrganizerView.vue

```diff
@@ -0,0 +1,142 @@
+<template>
+    <div class="st-view search-organization-view">
+        <STNavigationBar :title="t" />
```

Comment:
```suggestion
        <STNavigationBar :title="titlte" />
```
Je kan properties rechtstreeks aanspreken in je template, je hoeft geen extra variabel te maken